manifests/07-operator.yaml

```diff
@@ -53,3 +53,10 @@ spec:
               value: "cluster-image-registry-operator"
             - name: IMAGE
               value: docker.io/openshift/origin-docker-registry:latest
+          # volumeMounts:
```

Comment:
I assume you are going to remove these and just have them commented out for testing?

Reply:
@coreydaley Question on this one: the work being made on this PR is to mount the `trusted-ca` config map on the operand, not on the operator, right? If that is indeed the case then we gonna need these lines to be active in order to mount the config map also on operator during the deployment, right?

Reply:
@coreydaley yes once the CA controller lands

@ricardomaraschini this PR is built off of #340, which will mount `trusted-ca` to the operator. I did that because I also need to mount `trusted-ca` to the registry itself (operand)